vta/hardware/intel/Makefile

```diff
@@ -0,0 +1,82 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Directories
+ROOTDIR = $(CURDIR)
+BUILD_NAME = build
+BUILD_DIR = $(ROOTDIR)/../../$(BUILD_NAME)/hardware/intel
+SCRIPT_DIR = $(ROOTDIR)/scripts
+SRC_DIR = $(ROOTDIR)/../chisel
+TEST_DIR = $(ROOTDIR)/../../tests/hardware/common
+INCLUDE_DIR = $(ROOTDIR)/../../include
+
+# Executables
+TCLSH = quartus_sh
+
+# mode, options: skip_sim, sim

```

Comment:
We should remove this paragraph since these parameters are not being used in that Makefile

Reply:
Agree, I would remove the options that has not been used.